superscore/control_layers/core.py

```diff
@@ -98,7 +99,11 @@ def get(self, address: Union[str, Iterable[str]]) -> Union[EpicsData, Iterable[E
     @get.register
     def _get_single(self, address: str) -> EpicsData:
```

Comment:
My only suggestion is to update the return value in the signature here
```suggestion
    def _get_single(self, address: str) -> EpicsData | CommunicationError:
```

Reply:
Easy to forget, good catch